### PR TITLE
 [JBIDE-25700] added warning when using oc < 3.11 on linux

### PR DESCRIPTION
**Pull request by adietish**
# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by bdshadow** [comment]
@adietish according to Jenkins there are failures in `OCBinaryValidatorTest` and `ServerSettingsWizardPageModelTest`

**Comment by adietish** [comment]
@bdshadow yup, right, still, thx for pointing out

**Comment by adietish** [comment]
@bdshadow when testing, it is important to test how things behave in 3 places (they all had different versions of copy/pasted validation for the oc binary):
* Preferences
* Server adapter wizard
* New application wizard

**Comment by adietish** [comment]
@bdshadow @jkopriva I cannot reproduce the test errors in OCBinaryValidator on Mac. Will have to test on my Linux machine.

**Comment by adietish** [comment]
@bdshadow @jkopriva tests are fixed, plz review.
It is important to test how things behave in 3 places (they all had different versions of (copy/pasted) validation for the oc binary):
* Preferences
* Server adapter wizard
* New application wizard 

The warnings that are displayed are:
* oc not configured
* oc does not exist / is not executable
* linux, mac (not win): if path to oc contains spaces, oc >= 3.7 required
* linux: oc >= 3.11 is required (permission probs with versions below).

**Comment by jkopriva** [comment]
@adietish can you please also look at this warning in New OpenShift Connection Dialog? The warning text is cut.
image

**Comment by adietish** [comment]
@jkopriva thx, fixing...

**Comment by adietish** [comment]
@jkopriva @bdshadow both issues fixed. Please review.

File: plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/OpenShiftCoreMessages.properties [hunk 1]
```diff
@@ -12,13 +12,14 @@ DebugOnOpenshift=Debug on Openshift
 ProfileOnOpenshift=Profile on Openshift
 RunOnOpenshift=Run on Openshift
 
-OCBinaryLocationIncompatibleErrorMessage=OpenShift client version 1.1.1 or higher is required to avoid rsync issues. You may <a>download</a> and/or <a>configure</a> a newer version.
-OCBinaryLocationDontExistsErrorMessage=OpenShift client ({0}) does not exists. You may <a>download</a> and/or <a>configure</a> a new one.
-NoOCBinaryLocationErrorMessage=OpenShift client not configured. You may <a>download</a> and/or <a>configure</a> it.
+OCBinaryLocationIncompatibleErrorMessage=OpenShift client 'oc' version 1.1.1 or higher is required to avoid rsync issues. You may <a>download</a> and/or <a>configure</a> a newer version.
+OCBinaryLinuxRSyncPermErrorMessage=OpenShift client 'oc' version 3.11 is required to avoid permission errors when rsync'ing from a Linux client. You may <a>download</a> and/or <a>configure</a> a newer version.
```
Comment (bdshadow) [inline plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/OpenShiftCoreMessages.properties]: `rsync'ing` is it an intentional single quote?

Comment (adietish) [reply plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/OpenShiftCoreMessages.properties]: @bdshadow didnt think at how single quotes were avoinding param substitution. Luckily there are none behind that, but no, it's not intentional. Fixing.

**Pull request merged**